drivers/temperature/adt7420/adt7420.c

```diff
@@ -78,7 +78,7 @@ const struct adt7420_chip_info chip_info[] = {
  * @return register_value  - Value of the register.
 *******************************************************************************/
 int adt7420_reg_read(struct adt7420_dev *dev,
-		     uint8_t register_address, uint16_t *data)
+		     uint16_t register_address, uint16_t *data)
```

Comment:
can you explain in the commit description the reason for changing the size of the register address?

Reply:
The reason has now been mentioned in the commit message